test/tests/interfaces/pbs_sched_interface_test.py

```diff
@@ -0,0 +1,207 @@
+                self.assertTrue(
+                    'qmgr: Error (15211) returned from server' in e.msg[1])
+
+    def test_permission_on_scheduler(self):
+        """
+        Check for the permission to create/delete/modify scheduler object.
+        """
+        # Check for create permission
+        try:
+            self.server.manager(MGR_CMD_CREATE,
+                                SCHED,
+                                id="testCreateSched",
+                                runas=OPER_USER)
+        except PbsManagerError, e:
+            if self.server.get_op_mode() == PTL_CLI:
+                self.assertTrue(
+                    'qmgr: Error (15007) returned from server' in e.msg[1])
+
+        try:
+            self.server.manager(MGR_CMD_CREATE,
+                                SCHED,
+                                id="testCreateSched",
+                                runas=ROOT_USER)
+        except PbsManagerError, e:
+            self.fail(e.msg)
+
+        # Check for delete permission
+        self.server.manager(MGR_CMD_CREATE,
+                            SCHED,
+                            id="testDeleteSched")
+        try:
+            self.server.manager(MGR_CMD_DELETE,
+                                SCHED,
+                                id="testDeleteSched",
+                                runas=OPER_USER)
+        except PbsManagerError, e:
+            if self.server.get_op_mode() == PTL_CLI:
+                self.assertTrue(
+                    'qmgr: Error (15007) returned from server' in e.msg[1])
+
+        try:
+            self.server.manager(MGR_CMD_DELETE,
+                                SCHED,
+                                id="testDeleteSched",
+                                runas=ROOT_USER)
+        except PbsManagerError, e:
+            self.fail(e.msg)
+
+        # Check for attribute set permission
+        try:
+            self.server.manager(MGR_CMD_SET,
+                                SCHED,
+                                {'sched_cycle_length': 12000},
+                                id="TestCommonSched",
+                                runas=OPER_USER)
+        except PbsManagerError, e:
+            if self.server.get_op_mode() == PTL_CLI:
+                self.assertTrue(
+                    'qmgr: Error (15007) returned from server' in e.msg[1])
+
+        try:
+            self.server.manager(MGR_CMD_SET,
+                                SCHED,
+                                {'sched_cycle_length': 12000},
+                                id="TestCommonSched",
+                                runas=ROOT_USER)
+        except PbsManagerError, e:
+            self.fail(e.msg)
+
+    def test_delete_default_sched(self):
+        """
+        Delete default scheduler.
+        """
+        try:
+            self.server.manager(MGR_CMD_DELETE,
+                                SCHED,
+                                id="default")
```

Comment:
Just an idle comment.  There is a huge difference between using qmgr with '@default' and 'default', but they only have the one character difference between them.